src/gsModeling/gsParametrization.hpp

```diff
@@ -537,7 +537,7 @@ gsParametrization<T>::LocalParametrization::LocalParametrization(const gsHalfEdg
                 length = (*meshInfo.getVertex(indices.front()) - *meshInfo.getVertex(m_vertexIndex)).norm();
                 //length =  (meshInfo.getVertex(indices.front()) - meshInfo.getVertex(m_vertexIndex) ).norm();
                 nextAngle = angles.front()*thetaInv * 2 * EIGEN_PI;
-                nextVector = ((Eigen::Rotation2D<T>(nextAngle) * actualVector).normalized()*length) + p;
+                nextVector = ((Eigen::Rotation2D<T>(nextAngle).operator*(actualVector)).normalized()*length) + p;
```

Comment:
What's the underlying bug here?

Reply:
fixes this: https://cdash.ricam.oeaw.ac.at/viewBuildError.php?buildid=348488

Reply:
thanks. i still do not get it why this helps, but fine...